specification/2.0/Specification.adoc

```diff
@@ -1305,9 +1305,11 @@ Valid accessor type and component type for each attribute semantic property are
 
 The W component of each `TANGENT` accessor element **MUST** be set to `1.0` or `-1.0`.
 
+`COLOR_n` attributes with the `"VEC3"` type cannot be used with _unsigned byte_ or _unsigned short_ component types. As explained in section "3.6.2.4. Data Alignment", vertex attributes elements **MUST** be aligned to 4-byte boundaries inside a `bufferView`, but such configurations would result in elements 3 or 6 bytes in size. Padding would be required to satisfy alignment requirements, but that would result in the same amount of data as changing the accessor type to `"VEC4"`, therefore _unsigned byte_ or _unsigned short_ `COLOR_n` attributes **MUST** use `"VEC4"`, or alternatively, `"VEC3"` `COLOR_n` attributes **MUST** use _float_ component type.
```

Comment:
This change cannot be accepted. It's _a bit_ too late to stop requiring support for quantized VEC3 colors.